bindings/python/dlite-entity.i

```diff
@@ -493,6 +502,13 @@ in the storage plugin path.
 bool dlite_swig_has_instance(const char *id, bool check_storages=true);
 
 
+%feature("docstring", "\
+Returns a list of in-memory stored ids.
+
+") dlite_swig_istore_get_uuids;
+%rename(istore_get_uuids) dlite_swig_istore_get_uuids;
+char** dlite_swig_istore_get_uuids();
```

Comment:
This will leak the memory returned by `dlite_swig_istore_get_uuids()` when the wrapped. 

We need to tell SWIG that is has ownership of the memory and that it should free it when the corresponding python object goes out of scope. Can be done with the SWIG `%newobject` and `%delobject` directives.

I think it is fine to fix this in a separate PR to not delay this one unnessesary